setup_cfg_fmt.py

```diff
@@ -316,6 +316,42 @@ def _is_ok_classifier(s: str) -> bool:
     return [s for s in classifiers if _is_ok_classifier(s)]
 
 
+def _imp_classifiers(cfg: configparser.ConfigParser, setup_cfg: str) -> str:
+
+    TOX_TO_CLASSIFIERS = {
+        'py': 'Programming Language :: Python :: Implementation :: CPython',
+        'pypy': 'Programming Language :: Python :: Implementation :: PyPy',
+    }
```

Comment:
let's put this at the module scope (otherwise it gets built every time this function gets called)